crowdfunding_compassion/forms/project_creation_form.py

```diff
@@ -105,6 +107,14 @@ def compress(image):
             return new_image if bytes_len > new_bytes_len else old_image
         return old_image
 
+    def sanitized_url(self, values, key):
+        # Prevent the URL to be considered as a route if necessary
+        if values.get(key):
+            url = values[key]
+            if not url.startswith("http"):
+                return "http://" + url
```

Comment:
Should we use https by default? I think it's widely used nowadays...
```suggestion
                return "https://" + url
```

Reply:
It depends on what behaviour we want to achieve. Using `http://` the secured websites will automatically be redirected on `https://` and it will work normally for websites that are not secured. If we use by default `https://`, it will work normally for secured website but people will have a warning that they are going on an unsecured website if it is `http://` instead.
But `https://` should be the standard nowadays so maybe the second option is the best. Let me know what you think.

Reply:
I think the second option still encourages the good practices and we better warn or users if they leave a secure environment.

Reply:
Perfect, I added the modification